cli/azd/test/functional/testdata/snaps/aspire-full/AspireAzdTests.Web/manifests/containerApp.tmpl.yaml

```diff
@@ -18,15 +18,15 @@ properties:
       identity: {{ .Env.AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID }}
     secrets:
       - name: connectionstrings--cache
-        value: '{{ connectionString "cache" }}'
+        value: cache:6379
```

Comment:
FWIW - the move away from calling `connectionString` to just a fixed value (without any secrets) seems a little strange to me.  I'm guessing this is correct however and is an artifact of the refactoring we've been doing around who defines secrets and how they flow.

Reply:
the Redis cache changed to a `container.v0`.  For preview4, container.v0 can't be exposed with a public ingress, so there's no way to access the resource from outside the container environment. The connection string is also kwnon during creation time, as it only requieres the resource name and port:

<img width="657" alt="image" src="https://github.com/Azure/azure-dev/assets/24213737/951fda99-4b73-4040-b96b-0526ba6d21d4">

Note:  The value is still set a secret for the app container, but it is written during infra synth.